eternalcombat-api/src/main/java/com/eternalcode/combat/config/implementation/PluginConfig.java

```diff
@@ -196,7 +203,7 @@ public static class Messages extends OkaeriConfig {
         public String blockPlacingBlockedDuringCombat = "&cYou cannot place {MODE} {Y} coordinate during combat!";
 
         @Comment("# Message sent when player tries to enter a region")
-        public String cantEnterOnRegion = "&cYou can't enter on this region during combat!";
+        public String cantEnterOnRegion = "&cYou can't enter this region during combat!";
```

Comment:
I think the name `cantEnterToRegion` is more fitting.

Reply:
Thanks for review, this is a breaking change so it will be added in next major update